### PR TITLE
UI reformat code update buttons

### PR DESCRIPTION
**Pull request by Cxristian**
reformat all class in UI

### CODE REVIEW

File: src/main/java/com/jala/view/CellCheckBox.java [hunk 1]
```diff
@@ -22,10 +22,11 @@
...
 /**
  * Class CellCheckBox
+ *
  * @version 0.0.1.
  * @author: Luis Guardia
  */
-public class CellCheckBox extends DefaultCellEditor implements TableCellRenderer  {
+public class CellCheckBox extends DefaultCellEditor implements TableCellRenderer {
 
     private JComponent componentCheckBox = new JCheckBox();
```
Comment (AKatrin) [inline src/main/java/com/jala/view/CellCheckBox.java:31]: Add documentation please

File: src/main/java/com/jala/view/CellCheckBox.java [hunk 2]
```diff
@@ -81,6 +84,6 @@ public boolean stopCellEditing() {
     public Component getTableCellRendererComponent(JTable table, Object isCellSelected, boolean isSelected, boolean hasFocus, int row, int column) {
         if (isCellSelected == null)
```
Comment (Regis18) [inline src/main/java/com/jala/view/CellCheckBox.java]: add braces

File: src/main/java/com/jala/view/CustomRadioButton.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.jala.view;
+
+import javax.swing.*;
```
Comment (Regis18) [inline src/main/java/com/jala/view/CustomRadioButton.java]: You have to specify the importations

File: src/main/java/com/jala/view/JPanelConverterVideo.java [hunk 4]
```diff
@@ -20,27 +20,19 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.File;
+import javax.swing.*;
```
Comment (Regis18) [inline src/main/java/com/jala/view/JPanelConverterVideo.java]: specify importations

File: src/main/java/com/jala/view/CustomButton.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+package com.jala.view;
+
+import javax.swing.JButton;
+import java.awt.Color;
+import java.awt.Cursor;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+
+public class CustomButton extends JButton {
```
Comment (AKatrin) [inline src/main/java/com/jala/view/CustomButton.java:27]: Add documentation

File: src/main/java/com/jala/view/CustomTitleBorder.java [hunk 6]
```diff
@@ -12,13 +12,15 @@
 
 package com.jala.view;
 
-import javax.swing.*;
+import javax.swing.BorderFactory;
+
```
Comment (AKatrin) [inline src/main/java/com/jala/view/CustomTitleBorder.java]: remove empty line

File: src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java [hunk 7]
```diff
@@ -12,18 +12,16 @@
 
 package com.jala.view;
 
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
+import javax.swing.*;
```
Comment (AKatrin) [inline src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java]: use specific imports

File: src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java [hunk 8]
```diff
@@ -77,14 +75,16 @@ private void initComponent() {
...
+
+        JPanel pnlSouth = pnlPlayBtn();
+        this.add(pnlSouth, BorderLayout.EAST);
 
         JPanel jpForTable = pnlTableResult();
         this.add(jpForTable, BorderLayout.SOUTH);
 
-        JPanel pnlSouth = pnlPlayBtn();
-        this.add(pnlSouth, BorderLayout.WEST);
+
```
Comment (AKatrin) [inline src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java]: remove empty line

File: src/main/java/com/jala/view/CustomButton.java (hunk 5 above)
Comment (Cxristian) [reply src/main/java/com/jala/view/CustomButton.java:27]: ok

File: src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java (hunk 7 above)
Comment (Cxristian) [reply src/main/java/com/jala/view/JPanelSearchAdvancedVideo.java]: my IDE changes it only XD

**Pull request merged**